server.py

```diff
@@ -1,61 +1,76 @@
+import asyncio
+import json
+import logging
+import os
+import re
+import signal
+import sys
+import websockets
+
 from mcrcon import MCRcon
-import re, json, asyncio, websockets, os, signal, sys, logging
 
 RCON_HOST = os.getenv('RCON_HOST')
 RCON_PASSWORD = os.getenv('RCON_PASSWORD')
-RCON_PORT = os.getenv('RCON_PORT', default=25575)
-SOCKET_HOST = os.getenv('SOCKET_HOST', default='0.0.0.0')
-SOCKET_PORT = int(os.getenv('SOCKET_PORT', default=8888))
-LOG_LEVEL = os.getenv('LOG_LEVEL', default='ERROR')
-REFRESH_RATE = int(os.getenv('REFRESH_RATE', default=5))
+RCON_PORT = os.getenv('RCON_PORT', 25575)
+SOCKET_HOST = os.getenv('SOCKET_HOST', '0.0.0.0')
+SOCKET_PORT = int(os.getenv('SOCKET_PORT', 8888))
+LOG_LEVEL = os.getenv('LOG_LEVEL', 'ERROR')
+REFRESH_RATE = int(os.getenv('REFRESH_RATE', 5))
 
 logging.basicConfig(level=LOG_LEVEL)
 
 assert RCON_HOST is not None
 assert RCON_PASSWORD is not None
 
-async def sendLocations(websocket, path):
+
+async def send_locations(websocket, path):
     while True:
-        output = json.dumps(getLocations())
+        output = json.dumps(get_locations())
         try:
             await websocket.send(output)
-        except (websockets.exceptions.ConnectionClosedOK, websockets.exceptions.ConnectionClosedError, websockets.exceptions.ConnectionClosed):
+        except (websockets.exceptions.ConnectionClosedOK, websockets.exceptions.ConnectionClosedError,
+                websockets.exceptions.ConnectionClosed):
             await websocket.wait_closed()
-            logging.info("Client connection closed")
+            logging.info('Client connection closed')
             break
         finally:
             await asyncio.sleep(REFRESH_RATE)
 
-def getLocations():
+
+def get_locations():
     output = []
-    response = mcr.command("/list")
-    match = re.search(r"([0-9]+).+:(.+)", response)
-    numberOfPlayers = int(match.group(1))
+    response = mcr.command('/list')
+    match = re.search(r'([0-9]+).+:(.+)', response)
+    number_of_players = int(match.group(1))
 
-    if numberOfPlayers != 0:
-        playerList = match.group(2).strip().split(", ")
-        for playerName in playerList:
-            response = mcr.command("/data get entity {} Pos".format(playerName))
-            match = re.search(r"(\[.*\])", response)
-            playerPosition = eval(match.group(1).replace("d",""))
+    if number_of_players != 0:
```

Comment:
I mostly went after `pep8` violations, but usually the `!= 0` would be dropped, relying on the "truthiness" of the variable.

Reply:
Can you explain further what you mean here?